docs/static_site/src/_includes/get_started/linux/julia/build-from-source.md

```diff
@@ -1,2 +1,2 @@
-Refer to the [Julia section of the MXNet Ubuntu installation guide](/get_started/ubuntu_setup#install-the-mxnet-package-for-julia).
+Refer to the [Julia section of the MXNet Ubuntu installation guide](/get_started/ubuntu_setup.html#install-the-mxnet-package-for-julia).
```

Comment:
These routes are actually valid - no need to add `.html`. To get them to work as they do in production, enable the `modrewrite` extension in your `httpd` configuration and run your webserver with `httpd`. [Here](https://www.digitalocean.com/community/tutorials/how-to-rewrite-urls-with-mod_rewrite-for-apache-on-ubuntu-16-04) is a good guide for doing this on Ubuntu.

Reply:
> These routes are actually valid - no need to add `.html`. To get them to work as they do in production, enable the `modrewrite` extension in your `httpd` configuration and run your webserver with `httpd`. [Here](https://www.digitalocean.com/community/tutorials/how-to-rewrite-urls-with-mod_rewrite-for-apache-on-ubuntu-16-04) is a good guide for doing this on Ubuntu.

Thanks for commenting and sharing the useful guide! I've seen people sometimes use the .html extension and sometimes don't. Is there a particular reason behind it?

Reply:
Although either way will technically get the users to the right page, not using the extension is cleaner (we don't want to show `.html` in the URL bar when users are navigating the site because it looks messy). Better practice to use routing whenever possible.